bedrock/base/templates/includes/protocol/navigation/index.html

```diff
@@ -24,22 +24,12 @@
         {% if not hide_nav_download_button %}
         <div class="mzp-c-navigation-download">
           {{ download_firefox(alt_copy=_('Download Firefox'), dom_id='protocol-nav-download-firefox', button_color='mzp-t-secondary mzp-t-small', download_location='nav') }}
-          {% if LANG in ['en-US', 'de', 'fr'] %}
-            {% if LANG == 'de' %}
-              {% set fxa_button_text = 'Firefox-Konto erstellen' %}
-              {% set fxa_link_text = 'Das sind deine Vorteile' %}
-            {% elif LANG == 'fr' %}
-              {% set fxa_button_text = 'Créez un compte Firefox' %}
-              {% set fxa_link_text = 'Découvrez les avantages' %}
-            {% else %}
-              {% set fxa_button_text = 'Get a Firefox Account' %}
-              {% set fxa_link_text = 'Check out the Benefits' %}
-            {% endif %}
+          {% if LANG in ['en-US', 'en-CA', 'en-GB', 'en-ZA', 'de', 'fr'] %}
```

Comment:
We need to expose the new strings below for l10n. Once that happens, opening a PR to add the German and French transaltions to the [l10n repo](https://github.com/mozilla-l10n/www.mozilla.org/) should be easy enough. This feels like a better way to handle the existing translations, seeing as the ask is to open this up for all locales?